examples/run.py

```diff
@@ -8,7 +8,9 @@
 RATE_OF_TIME = 1.0  # Simulated seconds per wall-clock second
 
 categories = Simulator.list_scenario_categories()
+print(categories)
```

Comment:
If including this print statement in the PR please could you change it to `print(f"Categories: {categories}")` just so it's super obvious what's being printed?

Reply:
Removed! Was a developer's check